tool/src/webapp/signup/download/downloadSelections.jsp

```diff
@@ -28,6 +28,15 @@
 					<h:outputText value="#{msgs.events_attendee_download_instruction}" rendered="#{!DownloadEventBean.allowedToUpdate && DownloadEventBean.meetingsAvailable}" escape="false"/>
 					<h:outputText value="&nbsp;" escape="false"/>
 				</h:panelGrid>
+				<h:panelGrid columns="1">
+					<h:panelGroup>
+						<h:commandButton id="downloadEventsXlsTop"  action="#{DownloadEventBean.startXlsDownload}" value="#{msgs.event_download_xls_button}"  rendered="#{DownloadEventBean.meetingsAvailable}"/>

```

Comment:
All three buttons have 'Top' at the end of their names so that the IDs are different from the three buttons at the bottom of the jsp.